test/functional/apps/dashboard/_dashboard_save.js

```diff
@@ -86,4 +90,9 @@ bdd.describe('dashboard save', function describeIndexTests() {
 
     await PageObjects.common.clickCancelOnModal();
   });
+
+  // To avoid load errors in subsequent tests that refresh the index.
+  bdd.it('Finish each test on the landing page', async () => {
+    await PageObjects.dashboard.gotoDashboardLandingPage();
+  });
```

Comment:
Hmm.. Why make this an `it`? There is no expect here. Shouldn't this just be an `after` hook? Sounds like you want this to run at the end, so it would be an after all hook.